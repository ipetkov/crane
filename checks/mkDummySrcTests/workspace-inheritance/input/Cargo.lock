# This file is automatically @generated by Cargo.
# It is not intended for manual editing.
version = 3

[[package]]
name = "hello"
version = "0.2.0"

[[package]]
name = "print"
version = "0.2.0"
dependencies = [
 "hello",
 "world",
]

[[package]]
name = "world"
version = "0.2.0"
