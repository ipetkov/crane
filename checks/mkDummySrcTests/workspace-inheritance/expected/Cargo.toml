[workspace]
members = ["print"]

[workspace.package]
edition = "2021"
version = "0.2.0"
